packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -367,6 +366,16 @@ class SnackBar extends StatefulWidget {
   /// Called the first time that the snackbar is visible within a [Scaffold].
   final VoidCallback? onVisible;
 
+  /// Default vertical padding for this snack bar.
+  /// 
+  /// Defaults to 14.0.
+  /// 
+  /// This will not be overriden when setting [padding]'s top and/or bottom values.
+  /// Both paddings will just be added.
+  /// 
+  /// To ignore this padding, please set to 0.
```

Comment:
This comment does not seem necessary.

Reply:
I'll just wait for your response in this comment: https://github.com/flutter/flutter/pull/78133#discussion_r596321803
:bow: